Client/Frontend/Browser/Tabs/TabTrayV2ViewModel.swift

```diff
@@ -214,12 +214,13 @@ class TabTrayV2ViewModel: NSObject {
 
         let baseDomain = tab.sessionData?.urls.last?.baseDomain ?? tab.url?.baseDomain
         let urlLabel = baseDomain != nil ? baseDomain!.contains("local") ? " " : baseDomain : " "
-
-        cell.screenshotView?.image = tab.screenshot
+        // Setting default image as when when there is no screenshot
+        cell.screenshotView?.image = tab.screenshot ?? UIColor.white.image()
         // Set Favicon from domain url when screenshot is empty for tab
         if tab.screenshot == nil, let domainUrl = getTabDomainUrl(tab: tab) {
```

Comment:
Will this ever get hit now because of `?? UIColor.white.image()`

Reply:
Yeah it should get hit i.e. line 220 as we are only checking `tab.screenshot` but not `cell.screenshot`

Reply:
Maybe the other way around then? When will it ever stay white if this block is always going to change it again

Reply:
The block doesn't always gets executed. Ex. if there is no url say in case of blank tab then block will never get executed. This is the case which gets covered by the white image.

Reply:
Made it more clear